Frameworks/CoreLocation/CLLocationManager.mm

```diff
@@ -363,6 +384,16 @@ - (void)_handlePositionChangedEvent:(WDGGeolocator*)geolocator statusEvent:(WDGP
 }
 
 /**
+ * Handles compass's heading change events.
+ * @param {WDSCompass*} compass compass instance.
```

Comment:
The geolocator functions also have the same note, where they say {WDGGeolocator*} geolocator geolocator instance.

It looks pretty bad, but "{WDSCompass*} compass" is the type and name of the parameter, while "compass instance" is the description of what that parameter is. Would it make sense to add a dash between the name and description of all @param annotations?